addons/binding/org.openhab.binding.tplinksmarthome/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.openhab.binding</groupId>
+		<artifactId>pom</artifactId>
+		<version>2.2.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>org.openhab.binding.tplinksmarthome</artifactId>
+	<version>2.2.0-SNAPSHOT</version>
```

Comment:
remove this line (it is inherited from the parent)

Reply:
I'll remove it. I'll believe the plugin creation script generates this line, so I've seen it in many other binding pull requests.